scripts/vnet_route_check.py

```diff
@@ -50,7 +50,7 @@
 RC_ERR = -1
 default_vrf_oid = ""
 
-report_level = syslog.LOG_ERR
+report_level = syslog.LOG_WARNING
```

Comment:
i wouldn't recommend change the report_level as this may have backward compatibility issues